assets/javascripts/discourse/initializers/qa-edits.js

```diff
@@ -49,32 +88,40 @@ function initPlugin(api) {
           : 1;
 
       if (post.id === post.topic.postStream.stream[positionInStream]) {
-        if (topicController.replies_to_post_number) {
-          const commentsCount = post.topic
-            .get("postStream")
-            .postForPostNumber(
-              parseInt(topicController.replies_to_post_number, 10)
-            ).comments_count;
-
-          if (commentsCount > 0) {
-            result.push(
-              helper.h(
-                "div.qa-comments-count.small-action",
-                I18n.t("qa.comments_count", { count: commentsCount })
-              )
-            );
-          }
-        } else {
-          const answerCount = post.topic.answer_count;
-
-          if (answerCount > 0) {
-            result.push(
+        const answersCount = post.topic.posts_count - 1;
+
+        if (answersCount > 0) {
```

Comment:
To avoid one level of indentation on this already huge indentation...maybe either do an early return `if (answersCount <= 0) { return }` or just move this construction code into a separate function like `this._buildQAndAFilters(result, helper, answersCount, topicController)`. This gives me a heart attack :joy: 

![image](https://user-images.githubusercontent.com/920448/156509174-b3489c22-6756-49f6-b4a6-a3dd4a343960.png)

You could probably do early return on `if (post.id === post.topic.postStream.stream[positionInStream]) {` too

Reply:
haha sorry let me fix this :)